build_rules.gradle

```diff
@@ -593,6 +593,118 @@ ext.applyJavaNature = {
   }
 }
 
+// Reads and contains all necessary performance test parameters
+class JavaPerformanceTestConfiguration {
+  String pkbLocation = System.getProperty('pkbLocation')
+
+  String logLevel = System.getProperty('logLevel', 'INFO')
```

Comment:
Could you add single lines of comments explaining what these properties are ? For example, I'm not sure what 'official' is used for.

Reply:
ok